beprof/curve.py

```diff
@@ -31,8 +31,12 @@ class Curve(np.ndarray):
     Has additional field - info.
     """
 
-    def __new__(cls, input_array):
+    def __new__(cls, input_array, **meta):
         obj = np.asarray(input_array).view(cls).copy()
```

Comment:
Copying on the view here is also a subject of discussion. It doesn't necessarily need to stay here